pkg/reconciler/common/releases.go

```diff
@@ -73,9 +89,13 @@ func TargetManifest(instance v1alpha1.KComponent) (mf.Manifest, error) {
 // So we return the target manifest if status.version is empty.
 func InstalledManifest(instance v1alpha1.KComponent) (mf.Manifest, error) {
 	current := instance.GetStatus().GetVersion()
+	fmt.Println("the current is ")
```

Comment:
Is this DEBUG statements left by mistake... I know it’s WIP

Reply:
will remove it.